hype-submitter_2.11/src/main/scala/com/spotify/hype/package.scala

```diff
@@ -27,39 +25,22 @@ package object hype {
       model.ContainerEngineCluster.containerEngineCluster(project, zone, cluster)
   }
 
-  object GkeSubmitter {
-    def apply(stagingLocation: String, cluster: model.ContainerEngineCluster): Submitter =
-      setupShutdown(Submitter.create(stagingLocation, cluster))
-  }
-
-  object LocalSubmitter {
-    def apply(keepContainer: Boolean = false,
-              keepTerminationLog: Boolean = false,
-              keepVolumes: Boolean = false): Submitter =
-      setupShutdown(Submitter.createLocal(
-        dockerCluster(keepContainer, keepTerminationLog, keepVolumes)))
-  }
-
-  implicit class HFnOps[T](val hfn: HFn[T]) extends AnyVal {
-    def run(implicit submitter: Submitter, env: RunEnvironment): T = #!
-
-    def #!(implicit submitter: Submitter, baseEnv: RunEnvironment): T =
-      submitter.runOnCluster(hfn.run, hfn.env(baseEnv))
+//  object GkeSubmitter {
+//    def apply(stagingLocation: String, cluster: model.ContainerEngineCluster): Submitter =
+//      setupShutdown(Submitter.create(stagingLocation, cluster))
```

Comment:
I would use this `setupShutdown` wrapper for where we create GkeSubmitter and LocalSubmitter and remove the closeable interface from those.